io.sourceforge.tartube.Tartube.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id" : "com.github.axcore.tartube",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "tartube",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+    "--filesystem=home",
```

Comment:
this is a broad filesystem permission.

Reply:
Unavoidable, I'm afraid. 

Tartube is a video archiver that writes all files into a single directory. There are already checks in place to make sure files aren't written elsewhere by mistake. 

We can't guess in advance where users will want to put their video collection, so we have to grant them access to `home`.

The only other file written is the config file, written to XDG_CONFIG_HOME.

Reply:
`xdg-video` this should be where.

Reply:
From all the bug reports I've received, 99% of users want one of these download locations:

1. /home/USER/Desktop/MY-PRIVATE-VIDEO-FOLDER
2. /home/USER/MY-PRIVATE-VIDEO-FOLDER
3. An external hard drive with a much larger capacity 

If XDG_CONFIG_HOME is too broad, then perhaps it could be replaced with something that covers those three situations.

I have never seen anyone actually store their their videos in XDG_VIDEOS_DIR, so if that's the only acceptable permission, we are going to annoy a lot of people.